src/helper.js

```diff
@@ -0,0 +1,36 @@
+export const PlaneFieldsCheck = (
```

Comment:
functions should always start with small letter. Components should always start with capital letter

Reply:
file name should be something meaningful. 

PaperHelper.js 
> verify if paper is valid. will contain hooks and other functions for Paper component

EngineerHelper.js 
> verify if engineer is valid. will contain hooks and other functions for Engineer component

PlaneHelper.js 
> verify if plane is valid. will contain hooks and other functions for Plane component

Utils.js 
> should contain is name valid. will contain very generic functions like isNameValid